core/baseline-core-v1.0-psd01.md

```diff
@@ -3001,60 +3003,227 @@ This section of the document focuses on the requirements and considerations rela
+[[D10]](#d10) Testability: 
+
+Preconditions:
+
+* A PBI test system is up and running.
+* There is a given set of test BPI Subjects that are part of a test workgroup.
+* The test BPI participants in a workgroup have agreed to allow internal authoritative input data to a BPI workstep only from a single, specific data source.
+* The test workgroup is associated with only one test workstep.
+* The test workstep accepts only input data from the agreed upon data source for a given data schema, and the input data is transformed in an agreed upon manner to an expected output.
+* The agreed-upon input data source is configured with a test data set.
+* Another, second input data source is configured with the same test data set as the agreed-upon data source. 
+
+Test Steps:
+
+1. Verify that the input data provided to the BPI test workstep is from the agreed-upon authoritative data source.
+2. Verify that the input data complies with the established business logic of the test workstep.
+3. Verify that the BPI workstep processes the input data correctly.
+4. Switch the input data source, and repeat step 1.
+5. Verify that the workstep creates an error, and exits with an appropriate error message.
+
+Test Passing Criteria:
+
+* The first input data is verified to be from the agreed-upon authoritative source.
+* The BPI workstep processes the first input data correctly.
+* The BPI workstep creates an error, and exits with the expected error message, when the input data comes from the second input data source.
 
 #### **[CR36]>[O10]** 
 If the internal authoritative input data to a BPI workstep is single-sourced, that source MUST be authoritative.
 
-[[CR36]>[O10]](#cr36o10) Testability: An agreement on authoritative data sources will be embedded in a commercial document electronically represented on a BPI as required by [[R6]](#r6) to be bdining. And since [[R6]](#r6) is testable, [[CR36]>[O10]](#cr36o10) is testable.
+[[CR36]>[O10]](#cr36o10) Testability: 
+
+Preconditions:
+
+* A PBI test system is up and running.
+* There is a given set of test BPI Subjects that are part of a test workgroup.
+* The test workgroup is associated with only one test workstep.
+* There exists a system of record identified as the single source for the input data.
+* A test workstep has been created that requires internal authoritative input data from a single source.
+* The BPI Subjects involved in the workstep have agreed that the identified system of record is the authoritative source for the input data.
+* The test workstep accepts only input data from the agreed upon data source for a given data schema, and the input data is transformed in an agreed upon manner to an expected output.
```

Comment:
This is two preconditions, and the second is unnecessary for this requirement and redundant with other spec requirements

```suggestion
* The test workstep accepts only input data from the agreed upon data source for a given data schema.
```